packages/component-elife-app/server/authsomeMode.js

```diff
@@ -1,10 +1,12 @@
 const BluebirdPromise = require('bluebird')
+const { isAuthor } = require('./policies')
 
-// Policy type: (user: string, operation: string, object: any, context?: any) => boolean
+// Policy type: (user: string, operation: string, request: any, context?: any) => boolean
 const authsomePolicies = {
   // TODO: Somehow extend this list?
   isAuthenticated: userId => !!userId,
   isNotAuthenticated: userId => !userId,
+  isAuthor,
```

Comment:
I'm thinking maybe we change the policy name from what the user has to __be__ to what they are expected to be able to __do__. This policy will need extending soon to also return true is the user is a staff member. This will make the name of the policy inaccurate. My suggestion would be something like `hasAccessToManuscript` 

- see https://gitlab.com/hindawi/xpub/xpub-review/blob/develop/packages/app/config/authsome-mode.js#L97

Reply:
Done :+1: